src/olympia/abuse/templates/abuse/emails/ContentActionChangePendingRejectionDate.txt

```diff
@@ -0,0 +1,6 @@
+{% extends "abuse/emails/base.txt" %}{% block content %}
+
+As you are aware, your {{ type }} {{ name }} was manually reviewed by the Mozilla Add-ons team, at which point we found a violation of one or more Mozilla add-on policies.
```

Comment:
Just my read, but this sounds somewhat passive aggressive.. is the "As you are aware" necessary? I would think this type of email should be more matter of fact and not assume what people may or may not be aware of

Reply:
In my experience developers are very aware of rejections already :sweat_smile: - and I assume in most cases they are the ones asking for that date to be changed.

Regardless, this is the copy everyone approved (see [issue](https://github.com/mozilla/addons/issues/14920)) so I'm not in hurry to change it and go through validation with Operations, Product and Legal again.